wordpress-mobile/WordPress-Android
# [Blogging prompts v1] Hide "Remove from dashboard", hide share prompt button, decode HTML in prompt cards

Fixes #

To test:
- Enable blogging prompts feature flag

- Hide "Remove from dashboard"
    - Tap on "My Site" in bottom navigation
    - Tap on "HOME" tab
    - Scroll to prompt card
    - Tap on overflow menu
    - "Remove from dashboard" item should not be visible

- Hide share prompt
    - Tap on "My Site" in bottom navigation
    - Tap on "HOME" tab
    - Scroll to prompt card
    - "Share prompt" should never be visible

- Decode HTML in "My Site" card prompt
    - Change date to May 27th (a prompt that should be decoded)
    - Tap on "My Site" in bottom navigation
    - Tap on "HOME" tab
    - Scroll to prompt card
    - The prompt should be decoded correctly

- Decode HTML in compact card prompt
    - Change date to May 27th (a prompt that should be decoded)
    - Tap on "My Site" in bottom navigation
    - Tap on FAB
    - The prompt should be decoded correctly

- Hide "View more prompts"
    - Tap on "My Site" in bottom navigation
    - Tap on "HOME" tab
    - Scroll to prompt card
    - Tap on overflow menu
    - "View more prompts" item should not be visible

- Decode HTML in blogging prompt reminder notification
    - Change date to something before June 2nd (e.g. June 1st)
    - Tap on "My Site" in bottom navigation
    - Tap on "Site Settings"
    - Tap on "Reminders and prompts"
    - Enable reminders for Thursday and tap on "Include daily prompt"
    - Change date to June 2nd
    - The blogging prompt reminder notification should be decoded correctly

- Remove blogging prompts onboarding notification local scheduling
    - With the blogging prompts feature flag enabled, open the app
    - The blogging prompts onboarding notification should not be shown

## Regression Notes
1. Potential unintended areas of impact
None

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing

3. What automated tests I added (or what prevented me from doing so)
\-

PR submission checklist:
